mineinabyss-features/src/main/kotlin/com/mineinabyss/patreons/PatreonFeature.kt

```diff
@@ -27,8 +26,7 @@ import java.util.*
 @Serializable
 @SerialName("patreon")
 class PatreonFeature(
-    private val token: SerializableItemStack? =
-        LootyFactory.createFromPrefab(PrefabKey.of("mineinabyss", "patreon_token"))?.toSerializable()
+    private val token: SerializableItemStack? = CreationFunctions().newMittyToken()?.toSerializable()
```

Comment:
Dont make the factory class something you need to instantiate every time, just use a singleton object for simplicity